ds3_java_cli/src/main/java/com/spectralogic/ds3cli/views/cli/GetTapesWithFullDetailsView.java

```diff
@@ -33,7 +33,7 @@
     @Override
     public String render(final GetTapesWithFullDetailsResult obj) throws JsonProcessingException {
         final NamedDetailedTapeList result = obj.getTapesWithDetails();
-        if ((result == null) || (null == result.getNamedDetailedTapes())){
+        if ((result == null) || (null == result.getNamedDetailedTapes()) || (result.getNamedDetailedTapes().size() == 0 ) ){
```

Comment:
the null == in the middle condition sets off my OCD (backwards comparison) - I know you didn't change this... I may have even written as I used to prefer the constant to be on the left in comparisons until the PHB put the kibosh on that